config/crds/hive.openshift.io_clusterprovisions.yaml

```diff
@@ -5905,6 +110,7 @@ spec:
           - podSpec
           - stage
           type: object
+          x-kubernetes-preserve-unknown-fields: true
```

Comment:
Not sure why yet but when this is added, Kube no longer wants to let me apply the CRD:

```shell
~/go/src/github.com/openshift/hive remove-clusterprovision-podspec*
❯ k apply -f config/crds/hive.openshift.io_clusterprovisions.yaml
Warning: apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
The CustomResourceDefinition "clusterprovisions.hive.openshift.io" is invalid: spec.validation.openAPIV3Schema.type: Required value: must not be empty at the root


```

Reply:
It looks like adding x-kubernetes-preserve-unknown-fields to any v1beta1 CRD causes this error, it may not be supported until v1. 

This means we need to move from v1beta1 CRD generation to v1, which means we need to orphan the v3 OpenShift Hive shards from receiving updates, if we're going to fix this this way.